falcon/response.py

```diff
@@ -106,9 +106,12 @@ class Response(object):
             blocks as byte strings. Falcon will use *wsgi.file_wrapper*, if
             provided by the WSGI server, in order to efficiently serve
             file-like objects.
-        stream_len (int): Expected length of `stream`. If `stream` is set,
+
+        stream_len (int): Deprecated alias for :py:attr:`content_length`
```

Comment:
Let's go ahead and move the text following the deprecation notice (i.e., starting with "If stream is..." up into a `Note:` section under the docstring for the `content_length` property.

So, this would become simply (note the trailing period):

```py
stream_len (int): Deprecated alias for :py:attr:`content_length`.
```